portage/Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM busybox:latest
+FROM amd64:latest
 MAINTAINER Gentoo Container Team <containers@gentoo.org>
 
-ADD http://distfiles.gentoo.org/snapshots/portage-latest.tar.bz2 /
-RUN mkdir -p /usr && bzcat /portage-latest.tar.bz2 | tar -xf - -C /usr
+ENV PORTAGE_TARBALL portage-latest.tar.xz
```

Comment:
smallest and newest portage snapshot. Usually up to 7 days behind rsync'd portage will do just fine. If not, user have the ability to rsync on container start.